Silver table UseCase1

Ashwini-Patil1608 opened the pull request:
1. Performing an inner join between bronze_title_ratings and bronze_title_basics
2. Filter out only movies (titleType == 'movie'). 
3. Extract year, genres, average_Rating, num_Votes.

Ashwini-Patil1608 committed:
removing the backslash of 2nd table

Ashwini-Patil1608 committed:
removing backslash of all the table

Commit:
Added one comment

Ashwini-Patil1608 committed:
added silver notebook folder

Ashwini-Patil1608 committed:
silver file

Ashwini-Patil1608 committed:
updated ingest data

Commit:
Added Silver_Layer_UseCase1 Notebook

vamsi-simbus commented on src/Silver_Notebook/Silver_Layer_UseCase1_IMDB_Project.ipynb:
```diff
@@ -0,0 +1,198 @@
+   "source": [
+    "# Creating the Title_df DataFrame from the bronze_title_ratings table\n",
+    "title_df = spark.table('az_adb_simbus_training.imdb_project_schema.bronze_title_ratings')\n",
+    "title_df.limit(5).display()"
```
please comment the display statements

Ashwini-Patil1608 replied:
Commented the display command

vamsi-simbus commented on src/Silver_Notebook/Silver_Layer_UseCase1_IMDB_Project.ipynb:
```diff
@@ -0,0 +1,198 @@
+   "outputs": [],
+   "source": [
+    "#Checking the distinct values of the titleType column\n",
+    "joined_df.select('titleType').distinct().show()"
```
please handle null values in silver layer

Ashwini-Patil1608 replied:
Removed the null values from silver layer

vamsi-simbus reviewed (changes requested):
Please make the suggested changes

Commit:
Made Suggested Changes of Silver Layer Table

1. Comment out Display Command
2. Removed the null values
3. From StartYear Column extracted Year only and renamed it to Year
4. Filter out year is greater than 2000
5. From genres column, extract only the first genre
